sktime/transformations/base.py

```diff
@@ -10,12 +49,19 @@
     "_PanelToPanelTransformer",
 ]
 
+import warnings
 from typing import Union
 
 import numpy as np
 import pandas as pd
+from sklearn.base import clone
 
 from sktime.base import BaseEstimator
+from sktime.datatypes import check_is, convert_to, mtype, mtype_to_scitype
+from sktime.datatypes._series_as_panel import (
+    convert_Panel_to_Series,
+    convert_Series_to_Panel,
+)
```

Comment:
1. I didn't add this. This is stuff that has already been in the transformer base class file, so I left it.
Removing it carries two risks: (a) some reference use breaks; (b) someone becomes angry because I removed it.

2. Yes, we are - nothing in here uses the types. I think @abostrom wanted to update them at some points? The current ones are probably obsolete (I hope), but I don't want to remove it in this PR.